securedrop/tests/test_manage.py

```diff
@@ -14,6 +14,10 @@ def test_parse_args(self):
         # just test that the arg parser is stable
         manage.get_args()
 
+    def test_setup_verbose(self):
+        args = manage.get_args().parse_args(['run'])
+        manage.setup_verbosity(args)
```

Comment:
It's better with asserts indeed. But it has value without assert. If the covered code has a typo the test will fail. Without the test a typo would go unnoticed. In any case I added the required tests.